components/org.wso2.carbon.identity.conditional.auth.functions.analytics/src/test/resources/testng.xml

```diff
@@ -0,0 +1,34 @@
+<!--
+  ~ Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+  ~
+  ~ WSO2 LLC. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Identity-connector-analytics-functions-validation-test-suite">
+    <test name="call-analytics-nashorn" parallel="false">
+        <parameter name="scriptEngine" value="nashorn"/>
+        <classes>
+            <class name="org.wso2.carbon.identity.conditional.auth.functions.analytics.CallAnalyticsFunctionImplTest"/>
+        </classes>
+    </test>
+    <test name="call-analytics-graaljs" parallel="false">
```

Comment:
Conditionally run the graaljs related tests

Reply:
We can run these tests regardless of graaljs being a connector. Hence keeping these tests.